crates/core/Cargo.toml

```diff
@@ -38,3 +39,4 @@ snapbox = "0.4.14"
 [features]
 default = []
 derive = ["liquid-derive"]
+indexmap = ["dep:indexmap"]
```

Comment:
Should suffix the feature with a `2` to track that this feature is for v2 (and allow us to support a v3, etc)?